### PR TITLE
Bye Bye Prelogin submodule

### PR DESCRIPTION
**Pull request by biyeun**
👋 

@orangejenny // @nickpell  cc @gcapalbo

### CODE REVIEW

**Review by gcapalbo — APPROVED** [review]

**Comment by orangejenny** [comment]
Wow, this feels momentous - farewell prelogin!

File: corehq/apps/domain/templates/domain/select_plan.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
             {% endif %}
             <input type="hidden" name="plan_edition" data-bind="value: selected_edition">
 
-            {% include 'prelogin/_sections/software_services/shared_pricing_table.html' with postlogin=True %}
+            {% include 'accounting/partials/pricing_table.html' %}
```
Comment (nickpell) [inline corehq/apps/domain/templates/domain/select_plan.html]: Is this defined?

Comment (nickpell) [reply corehq/apps/domain/templates/domain/select_plan.html]: Heh, found it in your PR

**Review by nickpell — APPROVED** [review]

File: corehq/apps/domain/templates/domain/select_plan.html (hunk 1 above)
Comment (biyeun) [reply corehq/apps/domain/templates/domain/select_plan.html]: :wink:

**Pull request merged**